src/lepton/jpgcoder.cc

```diff
@@ -1951,7 +1951,11 @@ void process_file(IOUtil::FileReader* reader,
         case 0:
           if ( errorlevel.load() < err_tresh ) {
                 if (action == comp ) {
-                    fprintf(stderr, "%d %d\n",(int)ujgfilesize, (int)jpgfilesize);
+                    fprintf(stdout, "bulat %.6lf %s %d %d\n",
```

Comment:
Do we really need "bulat" prefix here?

Reply:
replaced with "STATS"

Reply:
I still have to add statistics scripts into the repository